README.md

```diff
@@ -31,11 +24,64 @@ The profiles controller uses [client-go library](https://github.com/kubernetes/c
 The details of interaction points of the profiles controller with various mechanisms from this library are
 explained [here](docs/controller-client-go.md).
 
+## Controllers
+
+The [cmd](./cmd) package contains source files for a variety of profile controllers for Kubeflow.
+
+For more information about profile controllers, see the documentation for the [client-go](https://github.com/kubernetes/client-go/) library, which contains a variety of mechanisms for use when
+developing custom profile controllers. The mechanisms are defined in the
+[tools/cache folder](https://github.com/kubernetes/client-go/tree/master/tools/cache) of the library.
```

Comment:
These point to the wrong repos

Reply:
Isn't that where the source example for creating a profile controller is found? Are we not using the client-go library then?

Reply:
Sorry I read quickly

"For more information about profile controllers"

The upstream repo doesn't refer to `Profile` controller, just generic controllers of a Foo object